NetCoreRestApi/BusinessLayer/Managers/CategoryManager.cs

```diff
@@ -4,16 +4,16 @@
 
 namespace BusinessLayer.Managers
 {
-    public class CategoryManager : ICategoryManager
+    public class CategoryManager<TId> : ICategoryManager<CategoryModel<TId>>
```

Comment:
Remove Tid from CategoryManager<TId>. CategoryManager should inherits generic BaseManager<> and ICategoryManager interface